src/components/ProjectSearch/TextSearch.js

```diff
@@ -70,6 +91,9 @@ export default class TextSearch extends Component {
     }
     this.focusedItem = null;
     this.props.searchSources(this.state.inputValue);
+
+    const updateCount = true;
```

Comment:
can we avoid storing this?

Reply:
I think we could signal the need to update in a way other than the flag.